packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [37.3.0]
+
+### Fixed
+
+- Use contract abi to decode the amounts of token balance changes ([#4775](https://github.com/MetaMask/core/pull/4775))
+- Change gas limit fallback ([#4739](https://github.com/MetaMask/core/pull/4739))
+- Add types for bridge txs ([#4714](https://github.com/MetaMask/core/pull/4714))
```

Comment:
This is an `Added` rather than a fixed I think.

Reply:
Done